pkg/ddl/ingest/config.go

```diff
@@ -63,10 +63,12 @@ func genConfig(ctx context.Context, memRoot MemRoot, jobID int64, unique bool, r
 	adjustImportMemory(ctx, memRoot, cfg)
 	cfg.Checkpoint.Enable = true
 	if unique {
-		cfg.TikvImporter.DuplicateResolution = lightning.DupeResAlgErr
+		if cfg.TikvImporter.Backend == lightning.BackendLocal {
```

Comment:
add-index always use local backend, maybe remove this. below too

Reply:
modified